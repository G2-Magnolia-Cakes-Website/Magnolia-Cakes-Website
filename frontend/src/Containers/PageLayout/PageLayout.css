.layout {
  width: 100%;
  display: flex;
  flex-direction: column;
  min-height: 100vh;
  justify-content: space-between;
}

.layout .business-name-wrapper {
  padding: 1rem 3rem 1rem 3rem;
  display: flex;
  align-items: center;
  gap: 1.5rem;
}

.layout .business-name-wrapper .logo {
  width: 100px;
  object-fit: contain;
  margin: 0;
}

.layout .business-name-wrapper h1.business-name {
  /* font-family: "Satisfy", cursive; */
  font-family: "Sacramento", cursive;
  font-size: 64px;
  margin: 0;
}

@media (max-width: 768px) {
  .layout .business-name-wrapper h1.business-name {
    font-size: 58px;
  }
}

@media (max-width: 480px) {
  .layout .business-name-wrapper {
    padding: 0.5rem 1rem 0.5rem 1rem;
    display: flex;
    align-items: center;
    gap: 1.5rem;
  }

  .layout .business-name-wrapper .logo {
    min-width: 100px;
  }

  .layout .business-name-wrapper h1.business-name {
    font-size: 24px;
  }
}
