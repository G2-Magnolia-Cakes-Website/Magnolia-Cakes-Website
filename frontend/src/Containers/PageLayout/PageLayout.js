import React from "react";
import Navbar from "../../Components/Navbar/Navbar";

const PageLayout = () => {
  return (
    <div>
      <Navbar />
    </div>
  );
};

export default PageLayout;
