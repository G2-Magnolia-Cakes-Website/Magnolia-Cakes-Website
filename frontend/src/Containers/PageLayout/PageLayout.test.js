import React from "react";
import renderer from "react-test-renderer";
import PageLayout from "./PageLayout";
import { BrowserRouter } from "react-router-dom";

describe("test PageLayout", () => {
  it("render PageLayout", () => {
    const wrapper = renderer
      .create(
        <BrowserRouter>
          <PageLayout />
        </BrowserRouter>
      )
      .toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
