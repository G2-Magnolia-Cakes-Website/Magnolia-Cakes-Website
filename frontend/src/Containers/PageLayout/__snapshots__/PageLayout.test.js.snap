// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test PageLayout render PageLayout 1`] = `
<div
  className="layout"
>
  <div
    className="header"
  >
    <nav>
      <div
        className="menu"
      >
        <div
          aria-expanded={false}
          className="hamburger-react"
          onClick={[Function]}
          onKeyUp={[Function]}
          role="button"
          style={
            Object {
              "cursor": "pointer",
              "height": "48px",
              "outline": "none",
              "position": "relative",
              "transition": "0.4s cubic-bezier(0, 0, 0, 1)",
              "userSelect": "none",
              "width": "48px",
            }
          }
          tabIndex={0}
        >
          <div
            style={
              Object {
                "background": "#efc7c7",
                "height": "3px",
                "left": "8px",
                "position": "absolute",
                "top": "13px",
                "transform": "none",
                "transition": "0.4s cubic-bezier(0, 0, 0, 1)",
                "width": "32px",
              }
            }
          />
          <div
            style={
              Object {
                "background": "#efc7c7",
                "height": "3px",
                "left": "8px",
                "opacity": "1",
                "position": "absolute",
                "top": "23px",
                "transition": "0.4s cubic-bezier(0, 0, 0, 1)",
                "width": "32px",
              }
            }
          />
          <div
            style={
              Object {
                "background": "#efc7c7",
                "height": "3px",
                "left": "8px",
                "position": "absolute",
                "top": "33px",
                "transform": "none",
                "transition": "0.4s cubic-bezier(0, 0, 0, 1)",
                "width": "32px",
              }
            }
          />
        </div>
      </div>
      <ul
        className=""
      >
        <li
          onClick={[Function]}
        >
          <a
            aria-current="page"
            className="active"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          onClick={[Function]}
        >
          <a
            className=""
            href="/gallery"
            onClick={[Function]}
          >
            Gallery
          </a>
        </li>
        <li
          onClick={[Function]}
        >
          <a
            className=""
            href="/online-store"
            onClick={[Function]}
          >
            Online Store
          </a>
        </li>
        <li
          onClick={[Function]}
        >
          <a
            className=""
            href="/location"
            onClick={[Function]}
          >
            Location
          </a>
        </li>
        <li
          onClick={[Function]}
        >
          <a
            className=""
            href="/workshop"
            onClick={[Function]}
          >
            Workshop
          </a>
        </li>
        <li
          onClick={[Function]}
        >
          <a
            className=""
            href="/about-us"
            onClick={[Function]}
          >
            About Us
          </a>
        </li>
        <li
          onClick={[Function]}
        >
          <a
            className=""
            href="/contact-us"
            onClick={[Function]}
          >
            Contact Us
          </a>
        </li>
        <li
          onClick={[Function]}
        >
          <a
            className=""
            href="/get-a-quote"
            onClick={[Function]}
          >
            Get a Quote
          </a>
        </li>
        <li
          onClick={[Function]}
        >
          <a
            className=""
            href="/cake-flavours-and-servings"
            onClick={[Function]}
          >
            Flavours & Servings
          </a>
        </li>
      </ul>
    </nav>
    <div
      className="signup-login-group"
    >
      <button
        className="login"
        onClick={[Function]}
      >
        Log In
      </button>
      <a
        className="signup"
        href="/signup"
        onClick={[Function]}
      >
        Sign Up
      </a>
    </div>
  </div>
  <div
    className="footer"
  >
    <img
      alt="Magnolia Cake Logo"
      className="logo"
      src="Magnolia_Cake_logo.png"
    />
    <div
      className="social-media-links-mobile"
    >
      <div
        className="social-media-wrapper"
      >
        <a
          href="https://www.facebook.com/magnolia.cake.au?mibextid=ZbWKwL"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="facebook"
            className="social-media-logo"
            src="facebook.svg"
          />
        </a>
        <a
          className="profile-name"
          href="https://www.facebook.com/magnolia.cake.au?mibextid=ZbWKwL"
          rel="noreferrer"
          target="_blank"
        >
          Magnolia Cake
        </a>
      </div>
      <div
        className="social-media-wrapper"
      >
        <a
          href="https://www.instagram.com/cakes_magnolia/?igshid=MzRlODBiNWFlZA"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="instagram"
            className="social-media-logo"
            src="instagram.svg"
          />
        </a>
        <a
          className="profile-name"
          href="https://www.instagram.com/cakes_magnolia/?igshid=MzRlODBiNWFlZA"
          rel="noreferrer"
          target="_blank"
        >
          cakes_magnolia
        </a>
      </div>
    </div>
    <div
      className="footer-sections-wrapper"
    >
      <div
        className="footer-section"
      >
        <h4>
          Our Location
        </h4>
        <p>
          Melbourne, VIC 3752
        </p>
        <a
          href="/location"
          onClick={[Function]}
        >
          More details
        </a>
      </div>
      <div
        className="footer-section"
      >
        <h4>
          Contact Us
        </h4>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </p>
        <a
          href="/contact-us"
          onClick={[Function]}
        >
          More details
        </a>
      </div>
      <div
        className="footer-section"
      >
        <h4>
          Terms & Conditions
        </h4>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        </p>
        <a
          href="/terms-and-conditions"
          onClick={[Function]}
        >
          More details
        </a>
      </div>
      <div
        className="social-media-links"
      >
        <div
          className="footer-section"
        >
          <h4>
            Follow Us
          </h4>
          <div
            className="social-media-wrapper"
          >
            <a
              href="https://www.facebook.com/magnolia.cake.au?mibextid=ZbWKwL"
              rel="noreferrer"
              target="_blank"
            >
              <img
                alt="facebook"
                className="social-media-logo"
                src="facebook.svg"
              />
            </a>
            <a
              className="profile-name"
              href="https://www.facebook.com/magnolia.cake.au?mibextid=ZbWKwL"
              rel="noreferrer"
              target="_blank"
            >
              Magnolia Cake
            </a>
          </div>
          <div
            className="social-media-wrapper"
          >
            <a
              href="https://www.instagram.com/cakes_magnolia/?igshid=MzRlODBiNWFlZA"
              rel="noreferrer"
              target="_blank"
            >
              <img
                alt="instagram"
                className="social-media-logo"
                src="instagram.svg"
              />
            </a>
            <a
              className="profile-name"
              href="https://www.instagram.com/cakes_magnolia/?igshid=MzRlODBiNWFlZA"
              rel="noreferrer"
              target="_blank"
            >
              cakes_magnolia
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
